Etabs_Adapter/CRUD/Read/Results/MeshResults.cs

```diff
@@ -271,6 +277,124 @@ private List<MeshResult> ReadMeshStress(List<string> panelIds, List<string> case
+            double[] s11Bot = null, s22Bot = null, s12Bot = null, sMaxBot = null, sMinBot = null, sAngBot = null, svmBot = null;
+            double[] s13Avg = null, s23Avg = null, sMaxAvg = null, sAngAvg = null;
+            double[] f11 = null, f22 = null, f12 = null, fMax = null, fMin = null, fAngle = null, fvm = null;
+            double[] m11 = null, m22 = null, m12 = null, mMax = null, mMin = null, mAngle = null; 
+            double[] v13 = null, v23 = null, vMax = null, vAngle=null;
+
+            List<MeshResult> results = new List<MeshResult>();
+
+
+            if (smoothing == MeshResultSmoothingType.ByPanel)
+                Engine.Base.Compute.RecordWarning("Stress values have been smoothed outside the API by averaging all force values in each node");
+
+            foreach (string caseName in cases)
+            {
+                m_model.Results.Setup.DeselectAllCasesAndCombosForOutput();
+                if (!SetUpCaseOrCombo(caseName))
+                    continue;
+
+                for (int i = 0; i < panelIds.Count; i++)
+                {
+
+                    List<MeshVonMises> stressVMTop = new List<MeshVonMises>();
+                    List<MeshVonMises> stressVMBot = new List<MeshVonMises>();
+                    int ret1, ret2, ret3;
+                    double panelThk;
+
+                    // Extract Von Mises Stresses
+                    ret1= m_model.Results.AreaStressShell(panelIds[i], itemTypeElm, ref resultCount, ref obj, ref elm, ref pointElm, 
+                        ref loadCase, ref stepType, ref stepNum, ref s11Top, ref s22Top, ref s12Top, ref sMaxTop, ref sMinTop, ref sAngTop, ref svmTop, 
+                        ref s11Bot, ref s22Bot, ref s12Bot, ref sMaxBot, ref sMinBot, ref sAngBot, ref svmBot, ref s13Avg, ref s23Avg, ref sMaxAvg, ref sAngAvg);
+                    
+                    // Extract Von Mises Resultant Axial Forces
+                    ret2 = m_model.Results.AreaForceShell(panelIds[i], itemTypeElm, ref resultCount, 
+                        ref obj, ref elm, ref pointElm, ref loadCase, ref stepType, ref stepNum,
+                        ref f11, ref f22, ref f12, ref fMax, ref fMin, ref fAngle, ref fvm, 
+                        ref m11, ref m22, ref m12, ref mMax, ref mMin, ref mAngle, 
+                        ref v13, ref v23, ref vMax, ref vAngle);
+
+                    // Get the panel thickness
+                    panelThk = GetPanelThickness(panelIds[i]);
+
+
+                    if ((ret1 == 0) && (ret2 == 0))
+                    {
+
+                        for (int j = 0; j < resultCount; j++)
+                        {
+                            int mode;
+                            double timeStep;
+
+                            // Calculate Von Mises Moment
+                            double Mvm = ComputeVonMisesMoment(svmTop[j], svmBot[j], panelThk);
```

Comment:
Should we be calculating moments here? If ETABS doesn't output the vonMises moment, I think we should just state that in the documentation and then the user can calculate the Mvm as they see fit.